src/executor/drop_executor.cpp

```diff
@@ -43,22 +43,48 @@ bool DropExecutor::DExecute() {
   const planner::DropPlan &node = GetPlanNode<planner::DropPlan>();
   DropType dropType = node.GetDropType();
   switch (dropType) {
+    case DropType::DB: {
+      auto database_name = node.GetDatabaseName();
+      auto current_txn = context_->GetTransaction();
+      if (node.IsMissing()) {
+        try {
+          auto database_object = catalog::Catalog::GetInstance()->GetDatabaseObject(
+              database_name, current_txn);
+        } catch (CatalogException &e) {
+          LOG_TRACE("Database %s does not exist.", database_name.c_str());
```

Comment:
Do we want to make this `LOG_ERROR` or `LOG_WARN`? Actually, we need to document what the correct error handling logic is here (and `CreateExecutor`). Should we just return false or should let the exception go up the stack.